README.md

```diff
@@ -225,6 +225,9 @@ The only way, from a dbt perspective, is to do a full-refresh of the incremental
 
 * Tables, schemas and database should only be lowercase
 
+* In order to avoid potential conflicts, make sure [`dbt-athena-adapter`](https://github.com/Tomme/dbt-athena) is not installed in the target environment.
```

Comment:
@nicor88 Once again, I am not totally sold on the wording so any suggestions are greatly appreciated 🙂